src/main/scala/uk/gov/homeoffice/drt/utils/DateUtil.scala

```diff
@@ -0,0 +1,39 @@
+package uk.gov.homeoffice.drt.utils
+
+import java.text.SimpleDateFormat
+import java.time.format.DateTimeFormatter
+import java.time.{LocalDate, LocalDateTime, ZoneId, ZoneOffset}
+import java.util.Date
+
+object DateUtil {
+
+  val parseDate: String => Date = date => new SimpleDateFormat("yyyy-MM-dd HH:mm:ss").parse(date)
+
+  val formatDate: Date => String = date => new SimpleDateFormat("yyyy-MM-dd HH:mm:ss").format(date)
+
+  val formatRequestDate: Date => String = date => new SimpleDateFormat("yyyy-MM-dd").format(date)
+
+  val parseRequestDate: String => Date = date => new SimpleDateFormat("yyyy-MM-dd").parse(date)
+
+  val isSameDate: (Date, Date) => Boolean = (date1, date2) =>
```

Comment:
`areEqual` might be better? We could probably also drop the `Date` part of all these vals since the object is already called `DateUtil`

Reply:
Usually we import DateUtil as class and use val directly in class so I think keeping Date in name would be nice . Also compare is api given by LocalDate so I think I would prefer to keep it .

Reply:
yes making this change